src/pages/workspace/accounting/reconciliation/CardReconciliationPage.tsx

```diff
@@ -26,12 +26,12 @@ function CardReconciliationPage({policy, route}: CardReconciliationPageProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
 
-    const [reconciliationConnection] = useOnyx(`${ONYXKEYS.COLLECTION.SHARED_NVP_EXPENSIFY_CARD_CONTINUOUS_RECONCILIATION_CONNECTION}${policy?.id}`);
-    const [isContinuousReconciliationOn] = useOnyx(`${ONYXKEYS.COLLECTION.SHARED_NVP_EXPENSIFY_CARD_USE_CONTINUOUS_RECONCILIATION}${policy?.id}`);
+    const [reconciliationConnection] = useOnyx(`${ONYXKEYS.COLLECTION.EXPENSIFY_CARD_CONTINUOUS_RECONCILIATION_CONNECTION}${policy?.workspaceAccountID}`);
+    const [isContinuousReconciliationOn] = useOnyx(`${ONYXKEYS.COLLECTION.EXPENSIFY_CARD_USE_CONTINUOUS_RECONCILIATION}${policy?.workspaceAccountID}`);
+    const [cardSettings] = useOnyx(`${ONYXKEYS.COLLECTION.EXPENSIFY_CARD_SETTINGS}${policy?.workspaceAccountID}`);
     const [bankAccountList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
-    const [cardSettings] = useOnyx(`${ONYXKEYS.COLLECTION.SHARED_NVP_PRIVATE_EXPENSIFY_CARD_SETTINGS}${policy?.id}`);
 
-    const paymentBankAccountID = cardSettings?.paymentBankAccountID ?? -1;
+    const paymentBankAccountID = cardSettings?.paymentBankAccountID ?? 0;
```

Comment:
Why do we need to change this line?

Reply:
yes, it caused some problems in another place, we want the value falsy (-1 is truthy)

Reply:
@koko57 Could you detail which problem is in other places?

Because we have a below rule 

https://github.com/Expensify/App/blob/98124799a639606573877bf6eb38554ee2939a22/contributingGuides/STYLE.md?plain=1#L474-L490

Reply:
but here you have a string, I wonder why we need to change it here as we have a number. And as I said - it caused a problem here https://github.com/Expensify/App/issues/45901#issuecomment-2262893111

Reply:
@koko57 

### 1. The current place (in this PR)

```
    const paymentBankAccountID = cardSettings?.paymentBankAccountID ?? 0;
    const bankAccountTitle = bankAccountList?.[paymentBankAccountID]?.title ?? '';
```

If we use 0 as a fallback value for `paymentBankAccountID`, when cardSettings?.paymentBankAccountID is undefined, paymentBankAccountID is 0 and bankAccountTitle is bankAccountList?.[0]

--> It mean that we always use bankAccountList?.[0] as default value


**_My suggestion:_** 
```
    const paymentBankAccountID = cardSettings?.paymentBankAccountID ?? -1;   // USING '-1'
    const bankAccountTitle = bankAccountList?.[paymentBankAccountID]?.title ?? '';
```

--> It mean that if cardSettings?.paymentBankAccountID is undefined, bankAccountTitle is empty string



### 2. Your case that you mentioned

Why do we need a fallback value in this case?

https://github.com/Expensify/App/blob/e6546d658b17a8dc94aee6e8ceba1eb988397053/src/pages/workspace/expensifyCard/WorkspaceExpensifyCardPage.tsx#L18

**_My suggestion:_** : We don't need fallback value 

```
    const paymentBankAccountID = cardSettings?.paymentBankAccountID;
```

Reply:
what is the difference here? bankAccountList?.[0] vs bankAccountList?.[-1]? bankAccountList is an object